### PR TITLE
Removes cloning (WAIT!, read the PR before downvoting!)

### PR DESCRIPTION
**Pull request by alexkar598**
> What the actual fuck alex are you cra- 

okay okay shut up for a second mr. contributor. Im not removing cloning fully, just from roundstart.
> THIS IS STUPID IM GOING TO DOWNVO-

calm down for a minute. Read the PR in its entirety then downvote it, ok?

# Please read the PR and my rational for the change before downvoting( or upvoting :) )/sending me colorful messages

### Let's start with the facts. This is what this PR does:
- On all maps but omega(omega is so small anyways lol, they need the cloner), the cloning pod, cloning scanner and cloning computer no longer spawn on roundstart
- On all maps, the cloning boards no longer spawn in tech storage
- Wherever there used to be a cloning setup, there is this now:

<details>
  <summary>Click here to see the in game paper detailing removal of cloning
</summary>

img

</details>

### Now, the why behind the how
This PR was created to solve several issues cloning introduces
- Doctors generally giving up easily on patients and simply throwing them into the clone pod
- Low skill cap in medical partly due to last point. This PR won't solve that issue but its a step in the right direction
- Death isn't really impactful when even if the traitor decapitated you and left your brain on the floor, you can easily be brought back to life within a few minutes at the cost of pretty much all your belongings.
- People simply dragging bodies to cloning and screaming at doctors when they try to do anything else with the body such as defibbing.

As for the benefits this PR brings to the table outside of current issues solved or partly solved:
- Actual meaningful interaction between departments, between medical and science for the research and while I'd love to say between medical and engineering, I don't have much hope for any significant cooperation between medical and engineering. Its probably going to be a lone bored med building a cloning setup by himself. Medically is really cooped up in their department and outside the one time upgrade to T4s, they dont get much interacting with the rest of the station outside healing the wounded.
- Gives the medical system slightly more depth. Its not much but its a step in the right direction.

Now this PR I think is great but it does come at downsides which cannot be overlooked:
- Due to the low skillcap and the resulting low attraction of players to medical. MDs are pretty incompetent overall. The removal of cloning certainly won't help getting players revived when they die and noone in the medical staff knows how to operate a defib without setting their johnson on fire. Unfortunatly, its a viscious circle, low depth in the medical system repel players from medical roles which in turn create a need for low depth medical systems. The loop has to be broken somewhere and this PR is part of the process of breaking the loop
- On lowpop box rounds, this is going to hurt. However, pod cloning and to a lesser extent, borging still exists and are viable revival methods. Hopefully we will be able to get more competant med players in the future which will make this point moot.

What I would like to see in future PRs altho I have little weight on what will or will not get accepted:
- Giving more depth to the medical system, perhaps with the concept of breaking bones, perhaps some other way
- Tying into the last point, making self serve medbay not a viable option for a quick complete recovery as individuals stealing items from vendors to heal themselves is a major issue while playing as MD.
- Making cloning more expensive to make it less viable to treat everyone using it and instead more a last resort when nothing else works and theyre FUBAR/theres 50 people everyone is dying oh god we need the HOS back alive ASAP. Perhaps using human meat alongside chemical additives as "fuel" for the cloner?
- Giving more reasons for departments to interact with each other instead of staying cooped up in their departments with their techfabs and catgirls.
- If this PR is merged and once it settles down, maybe remap things a bit to remove the empty room lol

I gave my best shot at giving a rational why removing cloning from the start of the round while still allowing it as an auxiliary option that can be built later on in the round but if that did not convince you, I suppose you will put a downvote on this PR.

:cl:  
rscdel: The station no longer starts with a functional cloning setup
/:cl:

### CODE REVIEW

**Comment by LoliconSlayer** [comment]
Too long didn't read, downvoting

**Comment by Darkstick** [comment]
I'm into it. Cloning stinks and makes death feel uneventful and mundane. Maybe if it required effort to get a cloning machine ready in the first place, it wont feel as lame.

**Comment by Jarod1200** [comment]
Need a way to fix husked miners earlier in the shift if you can't clone them

**Comment by PrinceKirze** [comment]
> Need a way to fix husked miners earlier in the shift if you can't clone them

Yeah early husks are gonna be a bitch to deal with.
However, I think I'm cautiously optimistic about this change. I'm pro the concept in and of itself but I'm wondering if our medical system (not the players) will be able to take up the slack when the players are ready to. I'm no medical main, so I can't give more meaningful feedback than that. I would have suggested a testmerge, but I think this change in particular would take so much time getting used to that it just needs to be tried out, and in a few months if somebody decides it was a real negative change it can be re-added.

File: code/modules/paperwork/paper_premade.dm [hunk 1]
```diff
@@ -10,6 +10,11 @@
 /obj/item/paper/fluff/shuttles/daniel
 	info = "i love daniel<br>daniel is my best friend<br><br>you are tearing me apart elise"
 	infolang = /datum/language/aphasia
+	
+/obj/item/paper/fluff/cloning
+	name = "notice- 'IMPORTANT'"
+	info = "<b>IMPORTANT NOTICE:</b><br>\nHey there Nanotrasen station! Due to budget cuts, we had to uninstall your cloning pod and sell it to shady aliens. Result: You have no cloning pod.<br>\n<br>\n<small><i>PS: You can still research and build a cloning pod yourself unlike your peers at GeeTea station.</i></small><hr><br>\n<small><i>PPS: Tell us if you spot any army of mutant insect clones. It turned out those shady aliens were doing shady business.</i></small>"
```
Comment (Hopekz) [inline code/modules/paperwork/paper_premade.dm:16]:
```suggestion
	info = "<b>IMPORTANT NOTICE:</b><br>\nHey there, Nanotrasen station! Due to budget cuts, we had to uninstall your cloning pod and sell it to shady aliens. Result: You have no cloning pod.<br>\n<br>\n<small><i>PS: You can still research and build a cloning pod yourself, unlike your GeeTea station peers.</i></small><hr><br>\n<small><i>PPS: Tell us if you spot an army of mutant insect clones. It turned out those shady aliens were doing shady business.</i></small>"
```

**Comment by Readystorm546** [comment]
Stealth M-ops nerf smh

**Comment by ghost** [comment]
@alexkar598 This is an okay solution but not a great one.

If you really want to go for the alternative, make cloning a body printer instead. 

If it just prints bodies, it makes for an excellent source of spare organs and limbs for medbay and for possible brain transplants to revive people as a more common practice, thus obligating some level of doctor involvement.

Otherwise I expect we'll still just see the cloner being rushed to be built and honestly since it'll be built more often then it'll likely lead to more clonepods than the standard, and clone-factories being made since someone will go above and beyond since they've already made the effort to build one clonepod.

**Comment by patpol4** [comment]
Finally someone did it

**Comment by Moggram** [comment]
I want more blood packs and iron pills at round stat if the cloning pods are going to be removed. A good number of the times a body comes in, it has been dragged all over the station. So after I get them all fixed up I have to mix 2-3 blood packs in them. Oh look, here comes a new body whit no blood and he is a Lizardperson?  Off to the morgue he goes because I only get one pack of L.

Also the cost of a Blood Pack Variety Crate in cargo is 3500. It is going to need to be dropped down to like... 2000-1500?
Or add more blood packs to it.

**Comment by patpol4** [comment]
Or just increase the amount of blood in blood packs with code

**Comment by alexkar598** [comment]
@Moggram
> 
> I want more blood packs and iron pills at round stat if the cloning pods are going to be removed. [snip]

Keep in mind its always possible to recover blood from corpses and that medical vendors start with a few bottle of saline solution which acts as a very good substitute instead of real blood as long as you can get the person to actually process the saline by staying alive for more than a split second.

Granted more blood packs would be nice but its out of scope of this PR sadly, shouldn't be too hard to make another PR for it later on

**Comment by alexkar598** [comment]
> If you really want to go for the alternative, make cloning a body printer instead.
> [snip]

That's a rather dramatic change which might as well remove cloning. My goal is to turn cloning into something else using small steps, not completly turn around the game instantly within a day, that's stupid and will lead nowhere as people will resist that change with alot of force.

Furthermore, I see no reason to make genetics even less useful to the medical staff. Currently, the medical staff has a dependency on genetics whenever they need a humanized monkey or something. I don't think removing that dependency is a good idea as it will just remove what little interaction genetics already has.

**Comment by ToGWtF** [comment]
You can dupe blood with chemistry

**Comment by MegaEmpirical** [comment]
I like it. I'd love to see a biomass component added but this is a nice start.

**Comment by N3D6** [comment]
Hope to see this merged, I like this PR.

**Review by patpol4 — CHANGES_REQUESTED** [review]
Just place the papers on tables so it doesnt lie on the ground like that with a lone chair next to it.
I'll probably remove the rooms and do something with the space later

**Comment by JamieD1** [comment]
Pointless. If people are too lazy to use defib then they will just get a cloner made.

TLDR: Solves nothing. And I'm not removing cloning entirely.

**Comment by alexkar598** [comment]
> TLDR: Solves nothing. And I'm not removing cloning entirely.

Someone didn't read my PR bruh

**Comment by alexkar598** [comment]
I don't have much to work with here but what you're complaining about:
- People building cloners during the round instead of defibbing
- You don't want to remove cloning

People will build cloners during the round, yes. Pretending they won't is stupid, the paper even hints at the fact they SHOULD build one. 
This serves two purposes:
1) Makes it an "upgrade" instead of simply something theyre supposed to be using all the time which opens it up to being tweaked further in the future to make it less shallow.
2) Encourage the medical department to get up and interact with others such as science and to a lesser extent, engineering to get a cloner setup

You dont want to remove cloning entirely:
I.... never talked about removing it entirely? It's a neat concept and has potential to be a good mechanic, it just kinda sucks it takes no effort to use and is in many ways superior to any other form of revival bar except if you want to get info out of a crime victim.

@JamieD1

**Comment by Moggram** [comment]
@alexkar598 

Could make a new PR that makes the cloning times a lot longer by two or three times?
And/or that it needs biomass form hydroponics (I think that this have been tied before and was shot down)

**Comment by alexkar598** [comment]
@Moggram That's pretty much a hard removal of cloning by making literally not viable for anything at all. I'm sorry but it's a terrible idea noone will like.

**Comment by Moggram** [comment]
> 
>it's a terrible idea noone will like.

And that is how space station 13 was stated.
Anyways I liked the PR, but I like bad stuff.
